src/popup/pages/SignTransaction.vue

```diff
@@ -8,16 +8,18 @@ import { IonPage } from '@ionic/vue';
 import { defineComponent, onMounted } from 'vue';
 import { useRoute } from 'vue-router';
 import { useI18n } from 'vue-i18n';
-import { Encoded } from '@aeternity/aepp-sdk';
+import {
+  buildTx, Encoded, Tag, unpackTx,
+} from '@aeternity/aepp-sdk';
 
-import { DEFAULT_WAITING_HEIGHT } from '@/constants';
+import { DEFAULT_WAITING_HEIGHT, PROTOCOLS } from '@/constants';
 import { RejectedByUserError } from '@/lib/errors';
 import { handleUnknownError } from '@/utils';
 import {
   useDeepLinkApi,
   useModals,
   useAeSdk,
-  useUi,
+  useUi, useAccounts,
```

Comment:
Please split this line.

Reply:
maybe a more opinionated formatter/linter should be used if those are implicit rules among developers